lib/ui/painting.dart

```diff
@@ -5591,7 +5607,21 @@ class Picture extends NativeFieldWrapperClass1 {
   ///
   /// To create a [Picture], use a [PictureRecorder].
   @pragma('vm:entry-point')
-  Picture._();
+  Picture._() { onCreate?.call(this); }
```

Comment:
You probably want to call this from `PictureRecorder.endRecording` instead of here, _after_ the native call to associate this object with the native peer.

Reply:
You can put it anywhere after this line: https://github.com/flutter/engine/blob/ebf65036d1bf0d7fefc55f6d1e21f61f85455752/lib/ui/painting.dart#L5764

That's the point where the native peer gets associated.

Reply:
(and then a call to `Picture.approximateBytesUsed` will work and not throw an exception)